insights/parsers/lsblk.py

```diff
@@ -270,3 +302,4 @@ def str2bool(s):
                     d[replace] = transform(d[original]) if transform else d[original]
                     del d[original]
             self.rows.append(BlockDevice(d))
+        self.device_data = {dev.name: dev for dev in self.rows}
```

Comment:
Please add this attribute to the docs.

Reply:
I've moved this documentation up into the BlockDevices as it now applies to both LsBlock and LsBlockPairs.